pkg/flypg/node.go

```diff
@@ -234,10 +270,98 @@ func (n *Node) PostInit(ctx context.Context) error {
 		if member != nil && member.Role != "" {
 			role = member.Role
 		}
+		fmt.Printf("My current role is: %s\n", role)
 
 		switch role {
 		case PrimaryRoleName:
-			// TODO - This is where we need to fence the primary if it's no longer valid.
+			standbys, err := repmgr.StandbyMembers(ctx, conn)
```

Comment:
still building my mental model here but shouldn't we query all members in the case that the local repmgr thinks the current "real" primary is now a stanby or vice-versa?

Reply:
Within this context, the local repmgr is identifying itself as the primary.

Reply:
ah right, thanks